tools/automatic_query_fixer/src/main/java/com/google/cloud/bigquery/utils/queryfixer/entity/FixOption.java

```diff
@@ -0,0 +1,16 @@
+package com.google.cloud.bigquery.utils.queryfixer.entity;
+
+import lombok.Value;
+
+@Value(staticConstructor = "of")
+public class FixOption {
+  /**
+   * A short description about this fixing option. Usually, if an identifier is to be modified, this
+   * field will be the new identifier to replace. For example, if a table is to be replaced, this
```

Comment:
nit: stylewise we usually don't use paragraph for field comment.
Just use "//..." for short comment.

Reply:
Done